src/index.html

```diff
@@ -1,13 +1,89 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Estonia&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+
+    <div class="Card" data-qa="card">
+      <img
+        class="Card-Image"
+        src="images/imac.jpeg"
+        alt="Product image">
```

Comment:
```suggestion
      <img
        class="Card-Image"
        src="images/imac.jpeg"
        alt="Product image"
      >
```
looks like code and demo links not consistent
on actual demo links image makes as a background.
please redeploy your solution.

Reply:
через VSD тести проходжу. Коли робив з background - не проходив (писав про це в слек)
Пробував видаляти демо лінк і повторно задеплоїти - не допомогло.
![image](https://github.com/mate-academy/layout_product-cards/assets/68586602/2b0175cc-d3b4-4ddd-81c2-7a43fdf83584)

Reply:
Передеплоїв. Наразі норм. Спочатку реалізовував через background і ніяк не міг пройти тести (писав про це в слек).

Reply:
you don't fix that
<img width="1078" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/8dce71c3-9e6b-4a1f-9392-26ae16f0ef8f">
<img width="738" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/2ca52cfd-4c0f-405d-86aa-d00abc241c3d">